tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -47,3 +49,21 @@ def get(self, request, *args, **kwargs):
         else:
             logger.info("Auth check FAIL for user on %s", timezone.now())
             return Response({"authenticated": False})
+
+class KibanaAuthorizationCheck(APIView):
+    """Check if user is authorized to view Kibana."""
+
+    query_string = False
+    pattern_name = "kibana-authorization-check"
+    permission_classes = [AllowAny]
```

Comment:
It's not a good practice to let open access, even a static generated token is more secure. For more detail, see https://www.django-rest-framework.org/api-guide/authentication/